src/http/cookie.cr

```diff
@@ -192,6 +192,14 @@ module HTTP
       end
     end
 
+    # Destroys the cookie.
+    # This is done by causing the cookie to expire. Also clears its value.
+    def destroy
+      self.value = ""
```

Comment:
I still do think that's a bit unexpected to clear the value in here. Just sayin'